source/clear-linux/tutorials/kata.rst

```diff
@@ -97,10 +87,32 @@ environment:
 You do not need to manually configure the runtime for Docker, because
 it automatically uses the runtime supported by the system.
 
-Check which runtime your system is using with the command:
+Troubleshooting
+===============
+
+To check which runtime your system uses, enter:
```

Comment:
In validating this command, I observe (on 28330) an error occurs: 

`WARNING: devicemapper: usage of loopback devices is strongly discourage for production use. 
      use `--storage-opt dm.thinpooldev` to specify a custom block storage device`

How would you suggest addressing this scenario? 

Note: When I run a typical `sudo docker info`, output is as expected.

Reply:
The default should be to use overlay.  https://docs.docker.com/storage/storagedriver/device-mapper-driver/  Docker is deprecating devicemapper support anyways.  You must have ```
{
  "storage-driver": "devicemapper"
} 
``` set in a config somewhere.

Reply:
Yes, this is what I had: 
{
 `storage-driver": "devicemapper"
}

Docker recommends using `overlay2` instead of `overlay`.  Can you please advise? 
If we change to either of these, I'll revise our Docker tut to reflect that change too (where I took it from):
https://clearlinux.org/documentation/clear-linux/tutorials/docker#id4

Reply:
I would follow Docker's recommendation.

Reply:
Please review my latest commit, above, for added note in docker.rst, and adding a link to it from kata.rst to docker.rst.

Reply:
Have you tested the daemon.json change and adding devicemapper?  I was trying out firecracker today with Kata which requires devicemapper and just adding this one setting prevented docker from restarting.  The issue was that if  /usr/lib/systemd/system/docker.service.d/clearlinux.conf had overlay2 specified and I added this daemon.json to /etc/docker with devicemapper then Docker failed to start.   If I changed /usr/lib/systemd/system/docker.service.d/clearlinux.conf to be devicemapper that worked and also worked if I had the same setting in that location and /etc/docker/daemon.json.   I didn't have time to look into this further to test and see if this is a Clear Linux bug or a misunderstanding in how to properly override files.  I think the proper way to override is in /etc/system/systemd/docker.service.d/<some conf> but that didn't work for me either or I had my file formatted wrong.

Reply:
Please see my latest commit, and let me know if you find this acceptable. 
I changed the storage driver in `etc/docker/daemon.json`  and it worked fine for docker.rst and kata.rst in this PR. I can't speak to firecracker. 

In kata.rst, this command produced no stdout when I ran it: `docker info | grep runtime`
I suggest that we remove it, unless you have an alternative suggestion.

Reply:
That works for me.  There might be an underlying issue when you try to change to devicemapper through this method but what you suggest should work. Devicemapper is being deprecated in Docker anyways.

Reply:
In response to your concern about devicemapper, in kata.rst, I added a link back to docker.rst, where I recommend using `overlay2`. There, I also provide links to Docker docs if users want to use other storage drivers. 

Will you please give a Y/N vote on this PR?

Reply:
Approved and LGTM